webpack/plugins/svg-compiler-plugin.js

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
```

Comment:
There are other loaders/plugins that do similar things, but not what we want.
We want to use `SVGSprite`. Even this one doesn't let you pass the configuration we need: https://www.npmjs.com/package/svg-sprite-loader, plus we want to run this loader in a webpack file not have requires inside of javascript files.